doc/articles/migrating-to-uno-sdk.md

```diff
@@ -0,0 +1,195 @@
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk.Web">
+  <Sdk Name="Uno.Sdk" />
+```
+
+## Uno Version
+
+The core libraries that ship from the main Uno repo (these are versioned together with Uno.WinUI), should now be controlled by the version of the Uno.Sdk that you are using. If you are using Central Package Management (there is a `Directory.Packages.props` in your solution) this will be very easy as you simply need to open the `Directory.Packages.props`. If you are not using Central Package Management you will need to open the various csproj files that have a Package Reference to one of the Uno NuGet's. You should do a Find/Replace for the version of Uno that you are using and replace it with `$(UnoVersion)`. The `$(UnoVersion)` property is set by the Uno.Sdk. To update Uno in the future simply update the version of the `Uno.Sdk` that you are targeting in the global.json as shown above.
+
+```xml
+<PackageVersion Include="Uno.UI.Adapter.Microsoft.Extensions.Logging" Version="$(UnoVersion)" />
+<PackageVersion Include="Uno.WinUI" Version="$(UnoVersion)" />
+<PackageVersion Include="Uno.WinUI.Lottie" Version="$(UnoVersion)" />
+<PackageVersion Include="Uno.WinUI.DevServer" Version="$(UnoVersion)" />
+<PackageVersion Include="Uno.WinUI.Skia.Gtk" Version="$(UnoVersion)" />
+<PackageVersion Include="Uno.WinUI.Skia.Linux.FrameBuffer" Version="$(UnoVersion)" />
+<PackageVersion Include="Uno.WinUI.Skia.Wpf" Version="$(UnoVersion)" />
+<PackageVersion Include="Uno.WinUI.WebAssembly" Version="$(UnoVersion)" />
+```
+
+## Cleaning up the Directory.Build.props
+
+There are several properties defined in the top of the `Directory.Build.props`. The values shown below are the default values from the previous templates. If these are still the values you have in your Directory.Build.props you can remove them.
+
+```xml
+<PropertyGroup>
+  <DebugType>portable</DebugType>
+  <DebugSymbols>True</DebugSymbols>
+
+  <DefaultLanguage>en</DefaultLanguage>
+  <IsAndroid>false</IsAndroid>
+  <IsIOS>false</IsIOS>
+  <IsMac>false</IsMac>
+  <IsMacCatalyst>false</IsMacCatalyst>
+  <IsWinAppSdk>false</IsWinAppSdk>
+</PropertyGroup>
+```
+
+> [!NOTE]
+> If you would like to continue to use the `Is{Platform}` properties in your project you should leave these in your Directory.Build.props along with the Choose/When block to set them when they are true.
+
+Beneath the PropertyGroup is a Choose block with conditions for various TargetFramework's. This can be entirely removed with one exception. By default the Sdk will provide the following values for you automatically setting the SupportedOSPlatformVersion and also the TargetPlatformMinVersion on Windows. If these values differ for your project be sure to keep your overrides. Any ItemGroup within the conditional blocks can be removed along with any other properties that are defined there.
```

Comment:
```suggestion
Beneath the `PropertyGroup` is a `Choose` block with conditions for various `TargetFramework`'s. This can be entirely removed with one exception. By default the Sdk will provide the following values for you automatically setting the `SupportedOSPlatformVersion` and also the `TargetPlatformMinVersion` on Windows. If these values differ for your project be sure to keep your overrides. Any `ItemGroup` within the conditional blocks can be removed along with any other properties that are defined there.
```